upstream-community-operators/federatorai/federatorai.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,484 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: federatorai.v0.0.1
+  namespace: placeholder
+  annotations:
+    capabilities: Basic Install
+    categories: OpenShift Optional
+    certified: "false"
+    repository: https://github.com/containers-ai/federatorai-operator
+    containerImage: prophetstor/federatorai-operator:0.0.1
```

Comment:
This doesn't match the containerImage used in the deployment section, which comes from `quay.io`

Reply:
Got it. We changed the repo to quay.io.